touchforms/backend/touchcare.py

```diff
@@ -212,9 +212,9 @@ def filter_cases(filter_expr, api_auth, session_data=None, form_context=None,
                  restore_xml=None, force_sync=True, uses_sqlite=False):
     session_data = session_data or {}
     form_context = form_context or {}
-    modified_xpath = "join(',', instance('casedb')/casedb/case%(filters)s/@case_id)" % \
-                     {"filters": filter_expr}
 
+    modified_xpath = "join(',', instance('casedb')/casedb/case%(filters)s[@status= 'open']/@case_id)" % \
+                     {"filters": filter_expr}
```

Comment:
does this just mean that it won't support submitting forms against closed cases? if so then i think that's fine. as a precaution, it's always good to run the manual sms test suite against anything we're not sure of